src/app/components/topnavigation/topnavigation.component.css

```diff
@@ -0,0 +1,51 @@
+.top-navigation {
+    left: 0;
+    top:0;
+    display: flex;
+    width: 100%;
+    position: fixed;
+    background-color: #1f1f1f;
+    justify-content: space-between;
+    align-items: center;
+    height: 50px;
+}
+
+.top-navigation__header-block {
+    align-items: center;
+    display: flex;
+    padding:  10px 10px 10px 10px;
+    gap: 5px
+}
+
+.top-navigation__name {
+    color: #fff;
+    text-transform: uppercase;
+    font-size: 22px;
+    font-family: 'Calibri';
+    letter-spacing: 2px;
+    font-weight: bold;
+}
+
+.top-navigation__cart {
```

Comment:
rename it cart-icon, else seems it would be styling about all of cart

Reply:
Changed